src/backend/tests/conftest.py

```diff
@@ -275,6 +276,97 @@ async def odk_project(db, client, project, tasks):
     yield project
 
 
+@pytest_asyncio.fixture(scope="function")
+async def submission(client, odk_project):
+    """Set up a submission for a project in ODK Central."""
+    odk_project_id = odk_project.odkid
+    odk_credentials = odk_project.odk_credentials
+    odk_creds = odk_credentials.__dict__
```

Comment:
I think this is a Pydantic model, so the way to dump would be `.model_dump()` 👍

Using __dict__ is never advised